sources/hubspot/__init__.py

```diff
@@ -84,17 +88,19 @@ def crm_objects(
     """Building blocks for CRM resources."""
     props = ",".join(_get_property_names(api_key, object_type))
     params = {"properties": props, "limit": 100}
+
+    yield from fetch_data(CRM_OBJECT_ENDPOINTS[object_type], api_key, params=params)
     if include_history:
-        params["propertiesWithHistory"] = props
-        # API allows max 50 items per call with property history
-        params["limit"] = 50
-    for objects, history_entries in fetch_data_with_history(
-        CRM_OBJECT_ENDPOINTS[object_type], api_key, params=params
-    ):
-        yield objects
-        yield dlt.mark.with_table_name(
-            history_entries, OBJECT_TYPE_PLURAL[object_type] + "_property_history"
-        )
+        # Get history separately, as requesting both all properties and history together
+        # is likely to hit hubspot's URL length limit
+        for history_entries in fetch_property_history(
```

Comment:
Changed this too. With some endpoints the URL was too long when listing all properties twice. :fearful: